client/app/scripts/liveblog-edit/controllers/blog-edit.js

```diff
@@ -113,12 +113,16 @@ define([
         // ask in a modalbox if the user is sure to want to overwrite editor.
         // call the callback if user say yes or if editor is empty
         function doOrAskBeforeIfEditorIsNotEmpty(callback, msg) {
```

Comment:
Why is `doOrAskBeforeIfEditorIsNotEmpty` expecting two parameters when you only send one?